src/gwt/src/org/rstudio/studio/client/workbench/UserInterfaceHighlighter.java

```diff
@@ -70,19 +91,109 @@ public Element getHighlightElement()
       {
          return highlightedElement_;
       }
-      
+
+      public void clearHandler()
+      {
+         if (handler_ != null)
+            handler_.removeHandler();
```

Comment:
Should we also set the handler to `null` after removing it?

Reply:
The handler is coded as a `final` variable so as is I can't reset it,  and `removeHandler()` is a no-op if called a second time. Do you think there is benefit for setting to `null` and removing the `final` keyword?

Reply:
Good point! In that case, perhaps not. (I mostly brought that up as it's a common pattern in GWT's own codebase)